### ISSUE TITLE
Weird app launcher visualization on Pop os

### ISSUE THREAD

**Issue by RePrete** [vinceliuice/WhiteSur-gtk-theme#374]
On latest Pop os 21.04 with the new app launcher there is a weird visualization. 
Selected app icon should have some text on the right, also I am currently using the dark version and the launcher is still clear.

Same thing for the alt + tab switch panel (I can't take a screen of this)

![Screenshot from 2021-07-25 14-13-08](https://user-images.githubusercontent.com/19281301/126898646-c9f2018c-6380-4181-8ceb-2f040a9f6832.png)

**Comment by Techno-Disaster** [comment]
Also for some reason my popups are white colored. Did I miss any flag?  @vinceliuice 
![image](https://user-images.githubusercontent.com/52817235/128946622-9f965b3f-42d4-45e6-8ad8-9038db80a70e.png)

**Comment by vinceliuice** [comment]
Don't you change the gnome-shell theme?

**Comment by Techno-Disaster** [comment]
> Don't you change the gnome-shell theme?

Yep my bad that fixed it. There was a warning saying remove user themes to fix authentication stuff so I removed that before

**Comment by Techno-Disaster** [comment]
> On latest Pop os 21.04 with the new app launcher there is a weird visualization.
> Selected app icon should have some text on the right, also I am currently using the dark version and the launcher is still clear.
> 
> Same thing for the alt + tab switch panel (I can't take a screen of this)
> 
> ![Screenshot from 2021-07-25 14-13-08](https://user-images.githubusercontent.com/19281301/126898646-c9f2018c-6380-4181-8ceb-2f040a9f6832.png)
> 
> Little update: look's like that the install command needed the root permissions to install gnome shell theme, at least for me.

This is still a issue though

**Comment by vinceliuice** [comment]
I'll try fix this on popos
